.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       name: Setup Stack
       with:
         enable-stack: true
-        stack-setup-ghc: true
+        stack-setup-ghc: false
```

Comment:
huh, does that change anything? if `haskell/actions/setup` doesn't ask stack to install ghc in this step, won't stack just install it at the beginning of the `stack build` step instead? I would expect the result to be the same...

Reply:
Seems not - actually I was trying to stop stack from pulling down ghc-9.0.1 unnecessary:
though hawk actually builds now with Nightly so we could actually build 9.0 with stack too.

Reply:
ooh, I see, it's downloading the ghc version corresponding to the lts specified in the `stack.yaml` file, not the one specified in the file we point to in `matrix.stack_yaml`! Let's repeat that change in a real PR, it should speed up CI.